### DISCUSSION TITLE
language settings

### DISCUSSION THREAD

**Discussion by xtjeyvc** [zim-desktop-wiki/zim-desktop-wiki#2668] [Q&A]
Brothers, I downloaded the 0.75 version of zim, but I can't find out where to modify the language settings. Does anyone know?

**Comment by dehesselle** [comment]
There is no UI setting for this. You can set the `LANG` variable before launching Zim, e.g. `LANG=en_US.UTF-8 zim`.

**Reply by sdsdesarrollo** [reply]
Hi..
Today a find where make changes
Zim 0.77.0

Edit environ.ini file in exe path
Uncomment the line below  # Interface language
<img width="311" height="88" alt="image" src="https://github.com/user-attachments/assets/de263aff-e984-4b5e-a123-2e926f94da71" />

**Comment by emailbin** [comment]
Unfortunately, I really don't know how to do this. Can you please explain it in more detail? Thank you very much!

**Reply by lorenrosen** [reply]
One thing: I assume zim uses the standard language for the system. Do you want a different language for zim, or more generally do you want to change the language for the whole system, including zim?

**Reply by emailbin** [reply]
I use German language for Lubuntu and I wanted to use English language only for zim.
